### ISSUE TITLE
 A11y_AzureNotebooks_Jupyter_MenuBar_Inspect: Role for Some of the Controls(File,Edit,View,Insert,Cell...etc) is incorrect.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3997]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Navigating through Menubar. 
4. Open Inspect Tool.
5. Verify Role for Controls(File,Edit,View,Insert,Cell...etc).

**Actual:**
Role for Controls(File,Edit,View,Insert,Cell...etc)is given as Hyperlink/Link which is incorrect.
 * Narrator is also reading as "File"Link which is incorrect.
 * NVDA is also reading as "File"Link which is incorrect.

**Note:**
Same issue is observed for all Controls present on menubar.

**Expected:**
Role for Controls(File,Edit,View,Insert,Cell...etc) Should be Menu.
 * Narrator should read as "File"Menu.
 * NVDA should read as "File"Menu.

**UserImpact:**
The user will not be able to know the significance of the control.

**Attachment:**
![a11y_azurenotebooks_jupyter_menubar_inspect](https://user-images.githubusercontent.com/39204909/45813540-8e472400-bcf0-11e8-9771-1ea45072a142.png)

**Comment by takluyver** [comment]
Closed by #4694
